docs/user-manual/en/using-server.md

```diff
@@ -50,6 +50,30 @@ The following highlights some important folders on the distribution:
 
 - `user-manual` - The user manual is placed under the web folder.
 
+## Artemis Shell
+
+You can activate the artemis shell by typing:
+
+```shell
+./artemis shell
+```
+
+This will start a JLine3 terminal with auto-completion:
```

Comment:
Sentence fragment, seems to end mid:

---
Also, this drop-in about the shell usage also doesnt really fit with the rest of the existing information on the page below that all assumes you are **not** using it. It could do with some elaboration / description linking and contrasting the different usages so the two routes are clear, and some new user actually following this page linearly isnt met with a sequence that essentially says 'do foo' followed by 'do this incompatible thing that requires you didnt do the earlier thing we already said'.

Reply:
A note on how to exit the shell may also be in order, some people wont actually know.

Reply:
unresolved as the docs didnt seem to change

Reply:
I removed the line.. and I'm adding a chapter for the CLI.